gedcomx-model/src/test/java/org/gedcomx/conclusion/PersonTest.java

```diff
@@ -57,91 +62,95 @@ public void testDisplayProperties() throws Exception {
     display.setName("9");
     person.setDisplayExtension(display);
     person = processThroughXml(person);
-    assertEquals("1", person.getDisplayExtension().getAscendancyNumber());
-    assertEquals("2", person.getDisplayExtension().getBirthDate());
-    assertEquals("3", person.getDisplayExtension().getBirthPlace());
-    assertEquals("4", person.getDisplayExtension().getDeathDate());
-    assertEquals("5", person.getDisplayExtension().getDeathPlace());
-    assertEquals("6", person.getDisplayExtension().getDescendancyNumber());
-    assertEquals("7", person.getDisplayExtension().getGender());
-    assertEquals("8", person.getDisplayExtension().getLifespan());
-    assertEquals("9", person.getDisplayExtension().getName());
+    assertThat(person.getDisplayExtension().getAscendancyNumber()).isEqualTo("1");
+    assertThat(person.getDisplayExtension().getBirthDate()).isEqualTo("2");
+    assertThat(person.getDisplayExtension().getBirthPlace()).isEqualTo("3");
+    assertThat(person.getDisplayExtension().getDeathDate()).isEqualTo("4");
+    assertThat(person.getDisplayExtension().getDeathPlace()).isEqualTo("5");
+    assertThat(person.getDisplayExtension().getDescendancyNumber()).isEqualTo("6");
+    assertThat(person.getDisplayExtension().getGender()).isEqualTo("7");
+    assertThat(person.getDisplayExtension().getLifespan()).isEqualTo("8");
+    assertThat(person.getDisplayExtension().getName()).isEqualTo("9");
 
   }
 
   @Test
   public void testPersonPersistentIdHelpers() throws Exception {
-    Person person = create();
+    final Person person = create();
     assertPersonEquals(person);
-    assertEquals(2, person.getIdentifiers().size());
-    assertEquals("pal", person.getPersistentId().toURI().toString());
+    assertThat(person.getIdentifiers().size()).isEqualTo(2);
+    assertThat(person.getPersistentId().toURI().toString()).isEqualTo("pal");
 
     person.setPersistentId(URI.create("urn:pal"));
-    assertEquals("urn:pal", person.getPersistentId().toURI().toString());
+    assertThat(person.getPersistentId().toURI().toString()).isEqualTo("urn:pal");
 
     person.getIdentifiers().clear();
-    assertNull(person.getPersistentId());
+    assertThat(person.getPersistentId()).isNull();
 
     person.setIdentifiers(null);
-    assertNull(person.getPersistentId());
+    assertThat(person.getPersistentId()).isNull();
 
     person.setPersistentId(URI.create("urn:pal"));
-    assertEquals("urn:pal", person.getPersistentId().toURI().toString());
+    assertThat(person.getPersistentId().toURI().toString()).isEqualTo("urn:pal");
+
+    final Person ref = person.persistentId(URI.create("urn:pal"));
+    assertThat(person.getPersistentId().toURI().toString()).isEqualTo("urn:pal");
+    assertThat(ref).isSameAs(person);
```

Comment:
this is the only real change to this class; the rest is migration to JUnit5 and AssertJ